api/WebKitCSSMatrix.json

```diff
@@ -50,10 +50,10 @@
             "version_added": true
           },
           "samsunginternet_android": {
-            "version_added": null
+            "version_added": "1.0"
           },
           "webview_android": {
-            "version_added": null
+            "version_added": "2"
```

Comment:
Just to be sure, is this a guess or based on testing WebView on Android 2?

Reply:
It's an educated guess based upon the WebKit version numbers -- I'm fairly confident this is accurate, but since these old WebView versions aren't much of a concern, I'm not too worried if it's off by a number or two.

Reply:
Ah, OK. Just checking that it wasn't accidentally copying the Chrome number.